src/vic3save/src/errors.rs

```diff
@@ -56,6 +56,9 @@ pub enum Vic3ErrorKind {
 
     #[error("invalid header")]
     InvalidHeader,
+
+    #[error("tag not found: {tag}")]
+    TagNotFound { tag: String },
```

Comment:
This error kind is a little awkward as it's only in `stats_cli` that it is raised, which really part of the library code. 

I would think an `.expect(format!("tag not found: {tag}"))` would be sufficient for the cli.